src/sources/video_card.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Credits: https://stackoverflow.com/a/49267844
+ */
+
+export interface VideoCard {
+  vendor: string
+  renderer: string
```

Comment:
The type says that these fields are strings. But one of your examples shows that the values can be numbers. So please transform the `getParameter` returned values to strings.

Reply:
added .toString() to both parameters

Reply:
The returned value of the `getParameter` method is `null` when the parameter is unknown. It will make `toString()` throw an error because this method doesn't exist in `null`. Please make the code handle this case. Setting the `VideoCard` field value to empty string is fine in this case.

Reply:
I added a failover empty string to avoid null
`vendor: (gl.getParameter(debugInfo.UNMASKED_VENDOR_WEBGL) || '').toString(),`
Instead of
`vendor: gl.getParameter(debugInfo.UNMASKED_VENDOR_WEBGL).toString(),`